docs/checkers/index.md

```diff
@@ -400,6 +400,25 @@ if True:  # Error was on this line
   pass
 ```
 
+### Use list literal (R1734) [](#R1734)
+
+This error occurs when list() is used instead of [] to create an empty list.
```

Comment:
Oh sorry I forgot to mention in my review, for any code embedded in text, use backticks (`) to make them format as code. This occurs for `list()` and `[]` in the text above.